src/main/webapp/resources/js/pages/analysis/components/Analysis.jsx

```diff
@@ -45,6 +45,12 @@ const analysisTypesWithAdditionalPage = [
 ];
 
 const analysisSettingsTabKeys = ["details", "samples", "share", "delete"];
+const analysisErrorTabKeys = [
+  "job-error-info",
+  "galaxy-parameters",
+  "standard-error",
+  "standard-out"
+];
 
 const Wrapper = styled.div`
   margin-left: ${SPACE_MD};
```

Comment:
On line 62, `analysisContextUpdateSubmissionName` is not used.

Reply:
Alread done.